cluster.go

```diff
@@ -66,7 +70,7 @@ func (t Endpoint) URI() string {
 	return fmt.Sprintf("%s:%d", t.Address, t.Port)
 }
 
-func (t C.qdb_remote_node_t) toStructG() Endpoint {
+func RemoteNodeToStructG(t C.qdb_remote_node_t) Endpoint {
```

Comment:
Right now we create a lot of utility methods like `RemoteNodeToStructG` and `TsBlobPointToStructG` and etc... So nature of this methods is strictly util. I would propose to move them to some external package. we can call it `apiutils`

Reply:
That works for me, these conversions are pretty much zero-overhead, right? 

What strikes me is that the Go API spends a lot of time doing all kinds of "proactive" conversions of everything into Go structures, which I have learned with the Python API and Java API is typically *not* the best way to approach things. E.g. in Java, we hold onto the raw C structure until the last possible moment. 

For these kind of RemoteNodeToStructG it's fine, because it's not a "hot" / "critical" code path, but for query results etc, this is super important.

Reply:
@solatis @vikonix on a second through I think it makes sense to keep methods as is in those packages. 
My bad that I'm still not well familiar with api - looks like those methods and overall packages true converters or encoders/decoders from C to Go and back. So it makes sense to keep methods there. 

Merging this will keep us going and will move api to latest Go compiler with minimal effort.